### PR TITLE
[JBIDE-22123] disable reset button when no parameter is selected

### PR DESCRIPTION
**Pull request by fbricon**
@adietish, @alexeykazakov care to review this respin-b candidate?

### CODE REVIEW

**Comment by jcantrill** [comment]
LGTM

**Comment by alexeykazakov** [comment]
+1

File: plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/newapp/TemplateParametersPage.java [hunk 1]
```diff
@@ -124,7 +124,13 @@ protected void doCreateControls(Composite container, DataBindingContext dbc) {
...
 		UIUtils.setDefaultButtonWidth(resetButton);
 		resetButton.addSelectionListener(onReset());
-
+		ValueBindingBuilder
+		.bind(WidgetProperties.enabled().observe(resetButton))
+		.notUpdatingParticipant()
+		.to(selectedParameter)
+		.converting(new IsNotNull2BooleanConverter())
+		.in(dbc);
+
```
Comment (adietish) [inline plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/newapp/TemplateParametersPage.java]: hey, @fbricon doing databinding? 1st I thought I'd not read correctly ;)

Comment (robstryker) [inline plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/newapp/TemplateParametersPage.java]: First you make him do something he doesn't want to... and then you rub it in his face and tell everyone.  Have you no common courtesy, Andre?  What about poor Fred's feelings.

Comment (fbricon) [inline plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/newapp/TemplateParametersPage.java]: Everyone chills now :-) I simply copied and pasted the code from the Edit button. I didn't have to start understanding how it works.

**Pull request merged**